CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+**0.7.0 - 09/08/23**
+ - Added generate_tax_1040 method
+ - Added copy_from_within_household noise function
+ - Improved performance of typographic noise function
+ - Added NO_NOISE option to dataset generation functions to aid in generating datasets without any noise
+ - Added support for python 3.11
+ - Sample data has been updated to reflect new data schema and simulation methods
```

Comment:
```suggestion
 - Sample population has been updated to reflect new data schema and simulation methods
```

Reply:
I suggested "sample population data" above, since we are talking specifically about updating the data